.gitignore

```diff
@@ -4,3 +4,4 @@
 .php_cs.cache
 composer.lock
 phpunit.xml
+.phpunit.result.cache
```

Comment:
should be done on dev-kit I guess

Reply:
I had a quick look, our `.gitignore` do not necessarily look like each other:

```
/build
/vendor
/docs/_build/*
.php_cs.cache
composer.lock
phpunit.xml

# clean up some non bower ready package
Resources/public/vendor/admin-lte/bootstrap
Resources/public/vendor/admin-lte/build
Resources/public/vendor/admin-lte/dist/img
!Resources/public/vendor/admin-lte/dist/img/boxed-bg.jpg # Only image called in AdminLTe.css
Resources/public/vendor/admin-lte/dist/js/app.js
Resources/public/vendor/admin-lte/dist/js/demo.js
Resources/public/vendor/admin-lte/dist/js/pages
Resources/public/vendor/admin-lte/documentation
Resources/public/vendor/admin-lte/pages
Resources/public/vendor/admin-lte/plugins
Resources/public/vendor/admin-lte/*.html
Resources/public/vendor/admin-lte/Gruntfile.js

Resources/public/vendor/iCheck/skins/all.css
Resources/public/vendor/iCheck/skins/*/*
!Resources/public/vendor/iCheck/skins/square/blue.css
!Resources/public/vendor/iCheck/skins/square/blue.png
!Resources/public/vendor/iCheck/skins/square/blue@2x.png
Resources/public/vendor/iCheck/icheck.js

Resources/public/vendor/slimScroll/examples
Resources/public/vendor/slimScroll/jquery.slimscroll.js

Resources/public/vendor/bootstrap/grunt
Resources/public/vendor/bootstrap/less
Resources/public/vendor/bootstrap/test-infra
Resources/public/vendor/bootstrap/Gruntfile.js

Resources/public/vendor/jqueryui/themes/black-tie
Resources/public/vendor/jqueryui/themes/blitzer
Resources/public/vendor/jqueryui/themes/cupertino
Resources/public/vendor/jqueryui/themes/dark-hive
Resources/public/vendor/jqueryui/themes/dot-luv
Resources/public/vendor/jqueryui/themes/eggplant
Resources/public/vendor/jqueryui/themes/excite-bike
Resources/public/vendor/jqueryui/themes/hot-sneaks
Resources/public/vendor/jqueryui/themes/humanity
Resources/public/vendor/jqueryui/themes/le-frog
Resources/public/vendor/jqueryui/themes/mint-choc
Resources/public/vendor/jqueryui/themes/overcast
Resources/public/vendor/jqueryui/themes/pepper-grinder
Resources/public/vendor/jqueryui/themes/redmond
Resources/public/vendor/jqueryui/themes/smoothness
Resources/public/vendor/jqueryui/themes/south-street
Resources/public/vendor/jqueryui/themes/start
Resources/public/vendor/jqueryui/themes/sunny
Resources/public/vendor/jqueryui/themes/swanky-purse
Resources/public/vendor/jqueryui/themes/trontastic
Resources/public/vendor/jqueryui/themes/ui-darkness
Resources/public/vendor/jqueryui/themes/ui-lightness
Resources/public/vendor/jqueryui/themes/vader

Resources/public/vendor/jquery/src

Resources/public/vendor/jquery.scrollTo/demo
Resources/public/vendor/jquery.scrollTo/tests
/build
/docs/_build
/vendor
.php_cs.cache
composer.lock
phpunit.xml
/vendor/
/.php_cs.cache
/composer.lock
/vendor
composer.lock
/build
/docs/_build
/vendor
.php_cs.cache
composer.lock
phpunit.xml
/build
/docs/_build
/vendor
.php_cs.cache
composer.lock
phpunit.xml
/build
/vendor
.php_cs.cache
composer.lock
phpunit.xml
/build
/docs/_build
/vendor
.php_cs.cache
composer.lock
phpunit.xml
/.phpunit.result.cache
/build
/vendor
/docs/_build/*
.php_cs.cache
composer.lock
phpunit.xml
/.phpunit.result.cache
/build
/docs/_build
/vendor
.php_cs.cache
composer.lock
phpunit.xml
/build
/docs/_build
/Resources/public/vendor/ionicons/png
/Resources/public/vendor/select2-bootstrap-css/*
!/Resources/public/vendor/select2-bootstrap-css/select2-bootstrap.min.css
/vendor
.php_cs.cache
composer.lock
phpunit.xml
/.sass-cache
/node_modules
/vendor
.php_cs.cache
composer.lock
phpunit.xml
/build
/docs/_build
/vendor
.php_cs.cache
composer.lock
phpunit.xml
/.php_cs.cache
/.env
/.bundle

###> symfony/framework-bundle ###
/.env.local
/.env.local.php
/.env.*.local
/config/secrets/prod/prod.decrypt.private.php
/public/bundles/
/var/
/vendor/
###< symfony/framework-bundle ###
/build
/vendor
.php_cs.cache
composer.lock
phpunit.xml
/.phpunit.result.cache
/build
/docs/_build
/vendor
.php_cs.cache
composer.lock
phpunit.xml
/.phpunit.result.cache
/build
/docs/_build
/vendor
.php_cs.cache
composer.lock
phpunit.xml
/build
/docs/_build
/vendor
.php_cs.cache
composer.lock
phpunit.xml
/.phpunit.result.cache
tests/Fixtures/App/app.sqlite
/build
/docs/_build
/vendor
.php_cs.cache
composer.lock
phpunit.xml
/docs/_build
/vendor/
.php_cs.cache
composer.lock
phpunit.xml
/.phpunit.result.cache
/docs/_build
/vendor/
.php_cs.cache
composer.lock
phpunit.xml
/.phpunit.result.cache
/build
/docs/_build
/vendor
.php_cs.cache
composer.lock
phpunit.xml
/.phpunit.result.cache
/docs/_build
/vendor
.php_cs.cache
composer.lock
phpunit.xml
/.phpunit.result.cache
/build
/vendor
.php_cs.cache
composer.lock
phpunit.xml
/build
/docs/_build
/vendor
.php_cs.cache
composer.lock
phpunit.xml
/build
/docs/_build
/vendor
.php_cs.cache
.phpunit.result.cache
composer.lock
phpunit.xml
.idea/
/vendor
/.php_cs
/.php_cs.cache

/node_modules/
npm-debug.log
yarn-error.log

/phpunit.xml
/build
/docs/_build/
/vendor
.php_cs.cache
composer.lock
phpunit.xml
/build
/docs/_build
/vendor
.php_cs.cache
composer.lock
phpunit.xml
/.sass-cache
/node_modules
/vendor
.php_cs.cache
composer.lock
phpunit.xml
/docs/_build/*
bin/doctrine-dbal
bin/doctrine-migrations
bin/php-parse
bin/yuicompressor.jar
build/
web/docs
app/config/parameters.yml
app/cache
app/logs
app/*.cache
web/config.php
web/bundles
web/uploads
.idea
.DS_Store
.env
doc-builder/docs
doc-builder/log
doc-builder/repositories
doc-builder/doctrees
vendor
!src/**/vendor
.sass-cache
.php_cs.cache

# composer related
composer.lock
composer.phar
vendor

# propel-generated stuff
Tests/Fixtures/App/app/propel/sql/
Tests/Fixtures/App/src/Sonata/TestBundle/Model/map
Tests/Fixtures/App/src/Sonata/TestBundle/Model/om
config/parameters.ini
app/cache
app/logs
app/*.cache
web/bundles
web/uploads

app/config/parameters*.yml
composer.lock
vendor
.idea
nbproject
composer.phar
/build
/docs/_build/
/vendor
.php_cs.cache
composer.lock
phpunit.xml
/build
/docs/_build/
/vendor
.php_cs.cache
composer.lock
phpunit.xml
/build
/docs/_build
/vendor
.php_cs.cache
composer.lock
phpunit.xml
/.phpunit.result.cache
/docs/_build
/vendor
.php_cs.cache
.phpunit.result.cache
composer.lock
phpunit.xml
/build
/docs/_build
/vendor
.php_cs.cache
composer.lock
phpunit.xml
/.phpunit.result.cache
```

Reply:
Can you please add this comment to: 
https://github.com/sonata-project/dev-kit/pull/582